### PR TITLE
Week 1 Homework

### PR DESCRIPTION
**Pull request by Mick26Soum**
Santy: Week 1 Homework

### CODE REVIEW

File: week1/homework/questions.txt [hunk 1]
```diff
@@ -1,6 +1,14 @@
 1. What is an object?
+  Objects are subgroups of a class.
```
Comment (reneedv) [inline week1/homework/questions.txt:2]: objects are instances of a class. I'm not sure what you mean by subgroups.

**Comment by reneedv** [comment]
Hi Santy,
  This looks good!
Thanks!
**Renée

File: week1/homework/questions.txt (hunk 1 above)
Comment (Mick26Soum) [inline week1/homework/questions.txt:2]: Hi Renee,

I meant to say that an object is an instance of a class. I wanted to say
something along the line.. for example a Large Blue, fast bike is an object
of the class Bike in a succinct way.

Couldn't find the words...I apologize for the confusion.

Thanks for checking!

**Comment by Mick26Soum** [comment]
Hi Renee,

Can you show me an example of how I could access my .bash file to
define/create a color reponse for the Rspec tests?

I'm currently getting a plain text response rspec.

I wrote down notes from the example that you showed in class but it doesn't
seem to work.

Thanks for your help.

**Pull request closed without merging**